Update project_setup.md (#208)
NLeSC/python-template: {{cookiecutter.project_name}}/project_setup.md

```shell
cd {{ cookiecutter.project_name }}
git init

> git add .
> git commit -m "first commit"
> git branch -M main
> git remote add origin {{ cookiecutter.repository }}
> git push --set-upstream origin main

Author: ```suggestion
git add .
git commit -m "first commit"
git branch -M main
git remote add origin {{ cookiecutter.repository }}
# on github, create {{ cookiecutter.repository }} as an empty repository 
git push --set-upstream origin main
```

```shell
cd {{ cookiecutter.project_name }}
git init

> git add .

Reviewer: ```suggestion
git add --all
```

Author: Good suggestion, thanks

of the
guide](https://guide.esciencecenter.nl/#/best_practices/language_guides/python?id=dependencies-and-package-management).

- Runtime dependencies should be added to `setup.cfg` in the `install_requires` list under `[options]`.

> - Development dependencies should be added to `setup.cfg` in the list under `[options.extras_require]`.

Author: ```suggestion
- Development dependencies should be added to `setup.cfg` in one of the lists under `[options.extras_require]`.
```

## Packaging/One command install

> You can distribute your code using PyPI or conda. Again, the project template does not enforce the use of either one.

Reviewer: ```suggestion
You can distribute your code using PyPI. Again, the project template does not enforce the use of either one.
```

> You can distribute your code using PyPI or conda. Again, the project template does not enforce the use of either one.

Reviewer: ```suggestion
You can distribute your code using PyPI.
```

  - Example tests that you should replace with your own meaningful tests (file: `test_my_module.py`)
- The testing framework used is [PyTest](https://pytest.org)
  - [PyTest introduction](http://pythontesting.net/framework/pytest/pytest-introduction/)
  - PyTest is listed as a development dependency, and can thus be installed with `pip3 install --editable .[dev]`
- Tests can be run with `pytest`
  - This is configured in `setup.cfg`

> - Uses [GitHub action workflows](https://docs.github.com/en/actions) to automatically run tests on GitHub infrastructure against multiple Python versions

Reviewer: ```suggestion
- The template uses [GitHub action workflows](https://docs.github.com/en/actions) to automatically run tests on GitHub infrastructure against multiple Python versions
```

  - Workflows can be found in [`.github/workflows`](.github/workflows/)
- [Relevant section in the guide](https://guide.esciencecenter.nl/#/best_practices/language_guides/python?id=testing)

## Documentation

> - Documentation should be put in the `docs/` directory. The contents have been generated using `sphinx-quickstart` (Sphinx version 1.6.5).

Reviewer: ```suggestion
- Documentation should be put in the [`docs/`](docs/) directory. The contents have been generated using `sphinx-quickstart` (Sphinx version 1.6.5).
```

## Continuous code quality

- [Sonarcloud](https://sonarcloud.io/) is used to perform quality analysis and code coverage report on each push
- Sonarcloud must be configured for the analysis to work
  1. go to [Sonarcloud](https://sonarcloud.io/projects/create)
  2. login with your GitHub account

>   3. add organization or reuse existing

Reviewer: ```suggestion
  3. add organization or reuse existing one
```
